### PR TITLE
PMM-6567 pg_stat_monitor.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Review by BupycHuk — CHANGES_REQUESTED** [review]
let's use disable_query_examples in tests

**Comment by JiriCtvrtka** [comment]
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1104

PRs:
PMM - https://github.com/percona/pmm/pull/584
PMM managed - https://github.com/percona/pmm-managed/pull/457
PMM API tests - https://github.com/Percona-Lab/pmm-api-tests/pull/124
PMM agent - https://github.com/percona/pmm-agent/pull/186

**Comment by AlekSi** [comment]
It should be `release/2.11`, not `release/2.11.0`

File: inventory/agents_test.go [hunk 1]
```diff
@@ -874,3 +874,344 @@ func TestPostgreSQLQanAgentExporter(t *testing.T) {
...
+					PMMAgentID:            pmmAgentID,
+					QueryExamplesDisabled: true,
+					CustomLabels: map[string]string{
+						"new_label": "QANPostgreSQLPgStatMonitorAgent",
+					},
+				},
+			},
+		}, getAgentRes)
+
+		// Test change API.
```
Comment (BupycHuk) [inline inventory/agents_test.go:1049]: let's remove change part for this test, because we already have it in the test above.

**Review by BupycHuk — APPROVED** [review]
@AlekSi it's my fault, I wrote wrong branch name to @JiriCtvrtka.
@JiriCtvrtka please create new branch and remove this one. Thank you.

**Review by askomorokhov — APPROVED** [review]

**Pull request merged**